aspnetcore/toc.md

```diff
@@ -100,6 +100,7 @@
 # [MVC](mvc/overview.md)
 ## [Razor Pages](xref:mvc/razor-pages/index)
 ### [Razor syntax](mvc/views/razor.md)
+### [Authorization conventions](mvc/razor-pages/razor-pages-authorization-conventions.md)
```

Comment:
Prefer an xref-style link.

Reply:
The TOC is full of `.md` links, too. When I opened the file, I just happened to land in a section of the file where `.md` links were dominant, so I thought I was just conforming to what's currently in the file. I'll fix this on the next commit.